react-native-purchases-ui/src/index.tsx

```diff
@@ -49,10 +52,12 @@ export default class RevenueCatUI {
   public static PAYWALL_RESULT = PAYWALL_RESULT;
 
   public static presentPaywall(): Promise<PAYWALL_RESULT> {
+    // leave this
     // TODO: check iOS/Android version
     return RNPaywalls.presentPaywall();
   }
 
+  // make this an object
```

Comment:
I think this is the last thing needed to ship this :D

Reply:
I will do it in another PR so the changes are more clear